app/main.py

```
@@ -31,12 +31,21 @@
 from app.modules.users.user_router import router as user_router
 from app.modules.users.user_service import UserService
 from app.modules.utils.firebase_setup import FirebaseSetup
+from contextlib import asynccontextmanager
+from app.core.dependencies import init_state, init_analytics_service
```

Ruff (0.8.2)

`app.core.dependencies.init_analytics_service` imported but unused

Remove unused import: `app.core.dependencies.init_analytics_service`

(F401)